inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -82,32 +83,48 @@ def contact_details(self, key, value):
     }
 
 
-@experiments.over('name_variants', '^419..')
-@utils.for_each_value
-def name_variants(self, key, value):
-    """Variants of the name."""
-    return value.get('a')
-
-
 @experiments.over('description', '^520..')
 @utils.for_each_value
 def description(self, key, value):
     """Description of experiment."""
     return value.get("a")
 
 
-@experiments.over('spokesperson', '^702..')
+@experiments.over('spokepersons', '^702..')
 @utils.for_each_value
-def spokesperson(self, key, value):
-    """Spokesperson of experiment."""
-    return value.get("a")
+def spokespersons(self, key, value):
+    """Spokepersons of the experiment."""
+    def _get_inspire_id(i_values):
+        i_value = force_single_element(i_values)
+        if i_value and isinstance(i_value, six.string_types):
+            type_, value = i_value.split('-')
+            return [
+                {
+                    'type': type_,
+                    'value': value,
+                },
+            ]
+
+    def _is_current(z_values):
+        z_value = force_single_element(z_values)
+        if z_value and isinstance(z_value, six.string_types):
+            return z_value.lower() == 'current'
+
+    return {
+        'name': force_single_element(value.get('a')),
+        'current': _is_current(value.get('z')),
+        'ids': _get_inspire_id(value.get('i')),
```

Comment:
So, resurrect the `_get_record` I just discarded? The only annoying thing is that I don't have test data...